GVFS/GVFS.FunctionalTests/Tools/GVFSProcess.cs

```diff
@@ -42,6 +49,27 @@ public bool TryMount(out string output)
             return this.IsEnlistmentMounted();
         }
 
+        public string AddIncludedFolders(params string[] folders)
+        {
+            return this.CallGVFS($"include {this.enlistmentRoot} -a {string.Join(";", folders)}");
+        }
+
+        public string RemoveIncludedFolders(params string[] folders)
+        {
+            return this.CallGVFS($" include {this.enlistmentRoot} -r {string.Join(";", folders)}");
+        }
+
+        public string[] IncludedFoldersList()

```

Comment:
From the above method names, I kindof expected `ListIncludedFolders` :)

Reply:
Changed to `GetIncludedFolders` since that seemed to fit better with current coding conventions.